Sources/LoggingTelegram/TelegramLogHandler.swift

```diff
@@ -60,10 +60,11 @@ public class TelegramLogHandler<T>: LogHandler where T: TelegramId {
 
   /// Creates a `TelegramLogHandler` for sending `Logger` output directly to Telegram.
   /// - Parameters:
-  ///   - label: The log label for he log handler.
-  ///   - chats: Target chats to send.
+  ///   - label: The log label for the log handler.
+  ///   - chat: Target chats to send to.
   ///   - token: Telegram Bot Token.
   ///   - level: (Optional) The minimal log level for this logger.
+  ///   - mute: (Optional)  Sending this message to the user silently
```

Comment:
The additional whitespace may be unnecessary?

Reply:
Sorry I made a mistake here... The second space is unnecessary. I'll fix it